src/components/UserInfo.tsx

```diff
@@ -0,0 +1,82 @@
+import { useSelector } from 'react-redux'
+import { getBearerSelector, getUserDataSelector } from '../store/selectors'
+import { Box, Stack, Tooltip, Typography, styled } from '@mui/material'
+import { RootState } from '../store/store'
+import SettingsIcon from '@mui/icons-material/Settings'
+import { useNavigate } from 'react-router-dom'
+import { UnknownResponseData } from '../types'
+
+export const UserInfo = () => {
+  const { snoovatar_size, snoovatar_img, name, total_karma, created_utc } =
+    useSelector<RootState, UnknownResponseData>(getUserDataSelector)
+  const bearer = useSelector(getBearerSelector)
+  const navigate = useNavigate()
+
+  const WhiteBoldTypography = styled(Typography)`
```

Comment:
Why do you use styled here?

Reply:
because it has different styling than usual Typography